### ISSUE TITLE
[BUG]: Filesystem module - change no_log to False for the `path` argument

### ISSUE THREAD

**Issue by Javedkhansiddiqui** [dell/ansible-powerscale#54]
**Describe the bug**

    - name: "Create Source Filesystem For DEV"
      dellemc.powerscale.filesystem:
        onefs_host: "{{ ip_address }}"
        port_no: "8080"
        verify_ssl: false
        api_user: "{{ api_user }}"
        api_password: "{{ api_password }}"
        path: "/ifs/{{ cluster_name }}/{{ region }}/DEV/{{ division }}{{ region }}{{ env }}{{ '{:>05}'.format(app_id) }}{{ '%02d' | format(item) }}"                        
        owner:
          name: '{{ api_user }}'
          provider_type: 'ads'
        state: "present"
      register: quota_output_dev_source
      when: not (quota_output_dev_source.changed | d(false))
      loop: '{{ list_of_sequence }}'
-----------
Hi Team,

i was able to successfully create a FileSystem using module which is not exist only
(eg: if nothing there it will create 1 , )
(eg: if 1 and 2 there it will create 3 , )
(eg: if 1, 2, 3 there it will create 4 , )

       #creating /ifs/cluster_name/MA/DEV/FOUMAD0031301
       #creating /ifs/cluster_name/MA/DEV/FOUMAD0031302
       #creating /ifs/cluster_name/MA/DEV/FOUMAD0031303

But i am not able to find the path which was created even with register option , it is "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER"
That's because the dellemc.powerscale.filesystem module declares `path`
as `no_log=True` in its argument specification:

[https://github.com/dell/ansible-powerscale/blob/1.9.0/plugins/modules/filesystem.py#L1565](url)

This tells ansible-core that the value of `path` is a secret that
should not be logged or accidentally returned.

**Expected behavior**
I am expecting to log the path which was created. like :- 
"path": "/ifs/cnpldcisit-hc01/MA/DEV/FOUMAD00313XX"

**Logs**
changed: [xxx.xxx.xx.org] => (item=3) => {
    "add_quota": "",
    "ansible_loop_var": "item",
    "changed": true,
    "create_filesystem": true,
    "delete_filesystem": "",
    "delete_quota": "",
    "filesystem_details": {
        "attrs": [
            {
                "name": "is_hidden",
                "namespace": null,
                "value": "False"
            },
            {
                "name": "size",
                "namespace": null,
                "value": "0"
            },
            {
                "name": "block_size",
                "namespace": null,
                "value": "8192"
            },
            {
                "name": "blocks",
                "namespace": null,
                "value": "64"
            },
            {
                "name": "last_modified",
                "namespace": null,
                "value": "Fri, 21 Apr 2023 12:55:40 GMT"
            },
            {
                "name": "change_time",
                "namespace": null,
                "value": "Fri, 21 Apr 2023 12:55:40 GMT"
            },
            {
                "name": "access_time",
                "namespace": null,
                "value": "Fri, 21 Apr 2023 12:55:40 GMT"
            },
            {
                "name": "create_time",
                "namespace": null,
                "value": "Fri, 21 Apr 2023 12:55:40 GMT"
            },
            {
                "name": "mtime_val",
                "namespace": null,
                "value": "1682081740"
            },
            {
                "name": "ctime_val",
                "namespace": null,
                "value": "1682081740"
            },
            {
                "name": "atime_val",
                "namespace": null,
                "value": "1682081740"
            },
            {
                "name": "btime_val",
                "namespace": null,
                "value": "1682081740"
            },
            {
                "name": "owner",
                "namespace": null,
                "value": "CS\\\\user"
            },
            {
                "name": "group",
                "namespace": null,
                "value": "Unknown Group"
            },
            {
                "name": "uid",
                "namespace": null,
                "value": "1718493"
            },
            {
                "name": "gid",
                "namespace": null,
                "value": "230"
            },
            {
                "name": "id",
                "namespace": null,
                "value": "10248563658"
            },
            {
                "name": "nlink",
                "namespace": null,
                "value": "2"
            },
            {
                "name": "type",
                "namespace": null,
                "value": "container"
            },
            {
                "name": "stub",
                "namespace": null,
                "value": "False"
            },
            {
                "name": "mode",
                "namespace": null,
                "value": "0700"
            }
        ],
        "namespace_acl": {
            "acl": [
                {
                    "accessrights": [
                        "dir_gen_read",
                        "dir_gen_write",
                        "dir_gen_execute",
                        "std_write_dac",
                        "delete_child"
                    ],
                    "accesstype": "allow",
                    "inherit_flags": [],
                    "op": null,
                    "trustee": {
                        "id": "SID:S-1-5-21-1229272821-706699826-839522115-11328533",
                        "name": "CS\\\\user",
                        "type": "user"
                    }
                },
                {
                    "accessrights": [
                        "std_read_dac",
                        "std_synchronize",
                        "dir_read_attr"
                    ],
                    "accesstype": "allow",
                    "inherit_flags": [],
                    "op": null,
                    "trustee": {
                        "id": "GID:230",
                        "name": null,
                        "type": null
                    }
                }
            ],
            "action": "replace",
            "authoritative": "mode",
            "group": {
                "id": "GID:230",
                "name": null,
                "type": null
            },
            "mode": "0700",
            "owner": {
                "id": "SID:S-1-5-21-1229272821-706699826-839522115-11328533",
                "name": "CS\\\\user",
                "type": "user"
            }
        }
    },
    "filesystem_snapshots": "",
    "invocation": {
        "module_args": {
            "access_control": null,
            "access_control_rights": null,
            "access_control_rights_state": null,
            "access_zone": "System",
            "api_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "api_user": "cs\\\\user",
            "group": null,
            "list_snapshots": false,
            "onefs_host": "xxx.xx.xxx.xxx",
            "owner": {
                "name": "cs\\\\user",
                "provider_type": "ads"
            },
            "path": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port_no": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "quota": null,
            "recursive": true,
            "recursive_force_delete": false,
            "state": "present",
            "verify_ssl": false
        }
    },
    "item": 3,
    "modify_filesystem": "",
    "modify_group": "",
    "modify_owner": "",
    "modify_quota": "",
    "quota_details": {
        "quotas": [],
        "resume": null
    }
}

**System Information (please complete the following information):**
 - OS/Version: [Fedora Linux 37 (Workstation Edition)]
 - Ansible Version [2.14]
 - Python Version [3.11]

**Comment by anupamaloke** [comment]
@Javedkhansiddiqui , thank you for submitting this issue. Just so you know, we are already working on this and it will be fixed as part of the 2.0.0 release. Meanwhile, as a workaround, you can just edit the `filesystem.py` module from the path where the dellemc.powerscale collection is installed (by default, the the module will be available in `~/.ansible/collections/ansible_collections/dellemc/powerscale/plugins/modules`), and change `no_log` to `False`:

```
1563: def get_filesystem_parameters():
1564:    return dict(
1565:       path=dict(required=True, type='str', no_log=False), 

```

**Comment by anupamaloke** [comment]
Closed by #56
